samples/bluetooth/peripheral_hr/src/main.c

```diff
@@ -115,11 +112,12 @@ void main(void)
 {
 	int err;
 
-	err = bt_enable(bt_ready);
+	err = bt_enable(NULL);
 	if (err) {
 		printk("Bluetooth init failed (err %d)\n", err);
 		return;
 	}
+	bt_ready();
```

Comment:
Empty line after `}`. I think I understand your thinking though:"this ties strongly into the preceding lines", however the style we follow is empty line after `}` unless the next one is also `}` (or e.g. `#endif`).